### PR TITLE
Add example readme

### PR DESCRIPTION
**Pull request by pollyndos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,99 @@
...
+
+For the `DxPopup` configuration in this example, refer to the following file: [ReportViewer.razor](BlazorReportViewer/Pages/ReportViewer.razor#L28).
+
+The [DxToastProvider](https://docs.devexpress.devx/Blazor/DevExpress.Blazor.DxToastProvider?v=24.1) component displays toast notifications to users. 
+For the `DxToastProvider` configuration, refer to the following file: [ReportViewer.razor](BlazorReportViewer/Pages/ReportViewer.razor#L73).
+
+### Email Service 
+
+> [!WARNING]  
+> In this example, we specify credentials for the authenticate to a SMTP server. In production, we recommend that you use [secrets](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-8.0&tabs=windows) to store sensitive information.
```
Comment (Kochkin) [inline README.md]: authentication

Comment (pollyndos) [reply README.md]:
```suggestion
> In this example, we specify credentials for the authentication to a SMTP server. In production, we recommend that you use [secrets](https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-8.0&tabs=windows) to store sensitive information.
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,101 @@
...
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# Product/Platform - Task
+# Reporting for Blazor - Email a Report from the Native Blazor Report Viewer
 
-This is the repository template for creating new examples. Describe the solved task here.
+This example uses the [Mailkit](https://mimekit.net/docs/html/Introduction.htm) library to send an email from our Native Blazor Report Viewer.
 
-Put a screenshot that illustrates the result here.
+The **Send Email** button in the Viewer’s toolbar opens the **Send Email** dialog ([DxPopup](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxPopup)). In the dialog,  you can specify the recipient list, subject, attachment, and body. Click the **Send** button to send the report with the specified settings:
```
Comment (dirk-pieterse) [inline README.md]:
```suggestion
The **Send Email** button in the Viewer’s toolbar opens the **Send Email** dialog ([DxPopup](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxPopup)). In the dialog,  you can specify the recipient list, subject, attachments, and body. Click the **Send** button to send the report with the specified settings:
```

File: README.md [hunk 3]
```diff
@@ -1,29 +1,101 @@
...
 
-Then, add implementation details (steps, code snippets, and other technical information in a free form), or add a link to an existing document with implementation details. 
+![Report Viewer - Send Email Window](images/send-email-window.png)
+
+The resulting email may look as follows:
```
Comment (dirk-pieterse) [inline README.md]:
```suggestion
An example of the resulting email:
```

File: README.md [hunk 4]
```diff
@@ -1,29 +1,101 @@
...
+
+![Report Viewer - Sent Email Example](images/sent-email-example.png)
+
+## Implementation Details
+
+### UI Elements
+
+The example handles the [`CustomizeToolbar`](https://docs.devexpress.com/XtraReports/DevExpress.Blazor.Reporting.DxReportViewer.OnCustomizeToolbar) event to add a **Send Email** button to the Viewer's Toolbar. The code snippet below adds locates the **Export To** command and adds the new button next to it:
```
Comment (dirk-pieterse) [inline README.md]: `adds locates` It adds or it locates?

File: README.md [hunk 5]
```diff
@@ -1,29 +1,101 @@
...
+        AdaptivePriority = 1,
+        Click = (args) => {
+            IsPopupVisible = ValidateEditingFields();
+            return Task.CompletedTask;
+        }
+    });
+}
+```
+
+A click on the newly added button opens a [DxPopup](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxPopup) where users can specify email options: recipients, subject, body. The popup form uses the following components:
```
Comment (dirk-pieterse) [inline README.md]:
```suggestion
A click on the newly added button opens a [DxPopup](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxPopup) where users can specify email options: recipients, subject, and body. The pop-up form uses the following components:
```

**Review by dirk-pieterse — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (pollyndos) [reply README.md]: @dirk-pieterse In this example, there can be only one file in the attachment. I am not sure about plural here.

File: README.md (hunk 4 above)
Comment (pollyndos) [reply README.md]:
```suggestion
The example handles the [`CustomizeToolbar`](https://docs.devexpress.com/XtraReports/DevExpress.Blazor.Reporting.DxReportViewer.OnCustomizeToolbar) event to add a **Send Email** button to the Viewer's Toolbar. The code snippet below locates the **Export To** command and adds the new button next to it:
```

**Pull request merged**